### PR TITLE
[DOC-155] Ed-Fi Exchange AWS Suite 3

### PR DESCRIPTION
**Pull request by roberthunterjr**
Migrates the Ed-Fi Exchange AWS Suite 3 Cloud Deployment folder from Confluence to Docusaurus.

[ConfluenceLink](https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22492394/Ed-Fi+ODS+API+Cloud+Deployment+for+AWS+Suite+3)

### CODE REVIEW

File: docs/getting-started/edfi-exchange/technology/ed-fi-ods-cloud-deploy-aws-suite-3/readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+:::
+
+## Description
+
+The Ed-Fi ODS / API deployment for Amazon Web Services consists of a set of Cloud Formation Templates and Lambda functions that make it easy to deploying a new copy of the Ed-Fi ODS / API to an Amazon Web Services tenant. The goal of these tools is to reduce the amount of time and effort needed to get a new instance of an Ed-Fi ODS/API up and running. This solution enables you to deploy the ODS / API on either PostgreSQL or Microsoft SQL Server.
+
+## Download
+
+* [Code & Documentation](https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22491023/Ed-Fi+ODS+API+Cloud+Deployment+for+AWS+-+Deployment+Guide)
```
Comment (stephenfuqua) [inline docs/getting-started/edfi-exchange/technology/ed-fi-ods-cloud-deploy-aws-suite-3/readme.md:15]: Update - provide `[Code]` link to GitHub repo and add link to the Deployment Guide here.

File: docs/getting-started/edfi-exchange/technology/learning-standards-sync/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Learning Standards Sync Utility
+
+## Overview
+
+The Ed-Fi Learning Standards Sync Utility is a tool for system administrators for synchronizing learning standards between the AB Connect API and a specified Ed-Fi ODS / API instance. Using the tool requires active credentials for both the target Ed-Fi ODS / API instance and the AB Connect API. The Sync Utility application provides a command-line interface (CLI) to set parameters and define options.
+
+## Download
+
+* [Code](https://github.com/Ed-Fi-Exchange-OSS/Ed-Fi-LearningStandards-Client)
+* [Documentation](https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22487416/Learning+Standards+Sync+Utility)
```
Comment (stephenfuqua) [inline docs/getting-started/edfi-exchange/technology/learning-standards-sync/readme.md:12]: Update link to point to the local Getting Started page

File: docs/getting-started/edfi-exchange/technology/learning-standards-sync/whats-new.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Learning Standards Sync Utility v 1.1 (Latest Release)
+
+The following improvements. and fixes were made in the Learning Standards Sync Utility 1.1 release.
+
+### Updates
+
+* The sync CLI command now supports retrieving only the Academic Benchmark standards that have changed during synchronization. This new behavior is enabled by default after an initial sync.
+* Server-side proxy updates were made to resolve learning standard sequencing issues between parent and child standards, which were causing download issues.
+  * The fix is documented in [EDFI-235](https://tracker.ed-fi.org/browse/EDFI-235?src=confmacro)
```
Comment (stephenfuqua) [inline docs/getting-started/edfi-exchange/technology/learning-standards-sync/whats-new.md:15]: Remove line 15

File: docs/getting-started/edfi-exchange/technology/ed-fi-ods-cloud-deploy-aws-suite-3/readme.md (hunk 1 above)
Comment (roberthunterjr) [reply docs/getting-started/edfi-exchange/technology/ed-fi-ods-cloud-deploy-aws-suite-3/readme.md:15]: Not sure what you mean here exactly. I added [Performance Test Code](./deployment-guide/readme.md) but I'm not sure what code this is suppose to be referencing.

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**